core/templates/services/exploration-html-formatter.service.spec.ts

```diff
@@ -164,6 +164,12 @@ describe('Exploration Html Formatter Service', () => {
     ).toBe(expectedHtmlTag);
   });
 
+  it('should fail for null interaction id', () => {
```

Comment:
`it should throw error when interaction id is null` would be better

Reply:
I modified the description now.